packages/terra-clinical-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Consumer } from 'xfc';
+import './EmbeddedContentConsumer.scss';
+
+const propTypes = {
+  /**
+   * The source URL of the content to load.
+   */
+  src: PropTypes.string.isRequired,
+  /**
+   * An object containing event handlers keyed by the event name.
+   * (e.g. {'eventA': function() {}, 'eventB': function() {}})
+   */
+  eventHandlers: PropTypes.object,
+};
+
+const defaultProps = {
+  src: undefined,
+  eventHandlers: {},
+};
+
+class EmbeddedContentConsumer extends React.Component {
+
+  componentDidMount() {
+    if (this.embeddedContentWrapper) {
+      Consumer.init();
+      this.xfcFrame = Consumer.mount(this.embeddedContentWrapper, this.props.src);
```

Comment:
Depending on how fancy we want to get out of the gate, we might want to expose all the Consumer mount options: https://github.com/cerner/xfc#setting-up-a-consumer

Basically it'd be adding secret and resizeConfig support.